package.json

```diff
@@ -13,7 +13,7 @@
         "prettier-watch": "onchange './**/*.{ts,tsx}' -- prettier --write {{changed}}",
         "lint": "eslint . ",
         "lint:fix": "eslint . --fix",
-        "ts-build": "tsc -b tsconfig.build.json",
+        "ts-build": "npm run -w nango copy:files && tsc -b tsconfig.build.json",
```

Comment:
it's a primitive (:D) script that only compiles typescript, it's cool that it's side-effects-free, what's the reason behind this?

Reply:
:+1: Removed in 141646cdc